CHANGES.md

```diff
@@ -1,3 +1,63 @@
+
+## [0.11.0] (2019-12-11)
+
+This is the first release since this repository was split off 
+from the [KMS](https://github.com/tendermint/kms) repo a few months 
+ago and contains more than the usual number of changes. 
+As the new repository matures we will be working towards a more robust 
+release cycle.
+
+This release also contains a first draft of the Tendermint Light Client :).
+
+The changes are organized in sections for better readability.
+
+Organizatinoal Changes:
+
+- Reorganized the crate into a workspace with a `tendermint` crate ([#30])
+- Remove all optional compilation ([#16])
+- Started using CircleCI for continuous integration ([#15])
+- Fix clippy lints ([#40], [#55])
+
+RPC Changes:
+
+- Fix `/commit` endpoint to actually include the commit data ([#42])
+- Use async/await for the rpc client ([#85])
+
+Type Changes:
+
+- Add `Default` trait impls and some other utilities to data types ([#64])
+- Fix transaction hash length to be 32-bytes ([#14])
+- Rename `LastCommit` to `Commit` ([#42])
+- Fix genesis file to include `validators` field ([#65])
+- Change `max_gas` from `u64` to `i64` ([#61])
+- Allow `Height` to be `0` ([#77])
+
+ABCI Changes:
+
+- Include `AbciQuery` in the `Method` enum ([#AbciQueryMethodEnum])
+- Fix deserializing ABCI Code field ([#13])
+- Fix ABCI data field to allow lower case hex encodings ([#17])
+- Fix `/abci_query` endpoint to take input `data` as hex and return `key`
+  and `value` in the response as base64 ([#77])
+
+Light Client:
+
+- Introduce validator `Set` type and compute Merkle root ([#6])
+- First draft implementation of logic for the light client ([#31, #36])
+
+- Dependency Changes:
+
+- Remove `secret_connection` and `ring` as dependencies (moved to KMS repo)
+  ([#60])
+- `tai64` from `2` to `3` ([#22])
+- `zeroize` from `0.9` to `1.1` ([#74, #89])
+- `hyper` from `0.10` to `0.13` ([#85])
+- `signatory` from `0.12` to `0.17` ([#89])
+- `subtle-encoding` from `0.3` to `0.5` ([#47])
+- `uuid` from `0.7` to `0.8` ([#91])
+- replace `rand_os` with `getrandome` ([#90])
+- pin to `ed25519-dalek` `1.0.0-pre.3` ([#83])
```

Comment:
This change was reverted by updating to signatory 0.17 (which now [does the pinning](https://github.com/tendermint/signatory/pull/185) for us). Do you still want to include it here?

Reply:
Yeah, I effectively reverted that and opened an upstream issue here (which has since been fixed): https://github.com/dalek-cryptography/ed25519-dalek/issues/108

Reply:
Duh thanks will remove.